control_plane/src/compute.rs

```diff
@@ -274,6 +274,8 @@ impl PostgresNode {
         conf.append("listen_addresses", &self.address.ip().to_string());
         conf.append("port", &self.address.port().to_string());
         conf.append("wal_keep_size", "0");
+        // walproposer panics when basebackup is invalid, it is pointless to restart in this case.
+        conf.append("restart_after_crash", "off");
```

Comment:
In case of PANICs (not errors) of other backend instance won't restart also, yes. Well, panic survival is a bit controversial subject, as panic in any case is not like 'normal' error, it means serious internal server problems. I think we could teach in walproposer to instruct postmaster kill the instance in case of wrong LSN problem and doesn't abuse restart_after_crash, but that needs some more consideration. For example, that would force local WAL redo after crash -- do we really want that? Console currently disables restart_after_crash for this reason.

Reply:
Ah, so the console already does that, and I thought that this PR is about console. Looks good then